pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -488,7 +488,8 @@ class Build {
                             target: "workspace/target/",
                             flatten: true)
 
-                    context.sh 'cd workspace/target/ && for file in $(ls *.tar.gz *.zip); do sha256sum "$file" > $fname.sha256.txt ; done'
+                    def extension = (buildConfig.TARGET_OS == "windows") ? "zip" : "tar.gz"
+                    context.sh "cd workspace/target/ && for file in \$(ls *.${extension}); do sha256sum \"\$file\" > \"\$file.sha256.txt\" ; done"
```

Comment:
This line looks somewhat unrelated. Perhaps something that should be upstreamed?